pkg/source/vmware/client.go

```diff
@@ -294,12 +296,9 @@ func (c *Client) IsPoweredOff(vm *migration.VirtualMachineImport) (bool, error)
 }
 
 func (c *Client) GenerateVirtualMachine(vm *migration.VirtualMachineImport) (*kubevirt.VirtualMachine, error) {
-	var boolFalse = false
-	var boolTrue = true
-
 	vmObj, err := c.findVM(vm.Spec.Folder, vm.Spec.VirtualMachineName)
 	if err != nil {
-		return nil, fmt.Errorf("error quering vm in GenerateVirtualMachine: %v", err)
+		return nil, fmt.Errorf("error querying vm in GenerateVirtualMachine: %v", err)
```

Comment:
per go doc, error wrapping is better with `%w` than `%v`

https://pkg.go.dev/fmt#Errorf

Reply:
Done